packages/cli/src/commands/env/pull.ts

```diff
@@ -125,6 +138,19 @@ export default async function pull(
     )}\n`
   );
 
+  output.print('\n');
+  if (deltaString) {
+    output.print(deltaString);
+  } else if (!oldEnv) {
+    output.log(
+      `Failed to find changes, but the \`${chalk.cyan(
+        filename
+      )}\` file was updated.`
```

Comment:
Should we say anything about failing in this case? I think we should only say `the .env file was updated`.

Reply:
I think you're right. The goal is to make the user aware that something went wrong if they were expecting a diff, but this is an awkward interaction. Will remove 👍